serviceroot.go

```diff
@@ -175,6 +178,7 @@ func (serviceroot *Service) UnmarshalJSON(b []byte) error {
 		Links              struct {
 			Sessions common.Link
 		}
+		Oem json.RawMessage // OEM message will be stored here
```

Comment:
I don't think we would need this because it can just be unmarshalled directly to the object's property. But no big deal, that can be cleaned up later.